autopush/tests/test_integration.py

```diff
@@ -619,6 +619,18 @@ def test_simplepush_data_delivery(self):
         yield self.shut_down(client)
 
     @inlineCallbacks
+    def test_disabled_simplepush(self):
+        state = self.conn.conf.disable_simplepush
+        self.conn.conf.disable_simplepush = True
+        try:
+            yield self.quick_register()
+        except AssertionError as ex:
+            # Check the raised assertion, since the true one is buried.
+            ok_("401 " in ex.message)
+            pass
```

Comment:
had to remove this test since I could not get a clean run without a reactor error.